frontend/packages/integration-tests-cypress/cy-command-additions/selectors.ts

```diff
@@ -0,0 +1,24 @@
+import { timeouts } from '../utils/timeouts';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      getByTestID(selector: string, timeout?: number): Chainable<Element>;
+      getByTestActionID(selector: string): Chainable<Element>;
+      getByLegacyTestID(selector: string, timeout?: number): Chainable<Element>;
+    }
+  }
+}
+
+// any command added below, must be added to global Cypress interface above
+
+Cypress.Commands.add('getByTestID', (selector: string, timeout: number = timeouts.default) =>
```

Comment:
```suggestion
Cypress.Commands.add('getTestID', (selector: string, timeout: number = timeouts.default) =>
```

Reply:
It's not getting a test id though, it's getting an element by test id.

Reply:
`getBy` doesn't clarify that, though. It has to be `getSomethingBySomething`, so `getElementByTestID`.

I still prefer `cy.testID('my-button').click()`. I think that reads well.

If we're having trouble finding a good name, we might stick with `cy.get('[data-test="my-button"]')` which is more explicit anyway.

Reply:
fixed, renamed to `byTestID`